src/libs/actions/FormActions.js

```diff
@@ -10,10 +10,10 @@ function setIsLoading(formID, isLoading) {
 
 /**
  * @param {String} formID
- * @param {String} error
+ * @param {Object} errors
  */
-function setErrorMessage(formID, error) {
-    Onyx.merge(formID, {error});
+function setErrorMessage(formID, errors) {
```

Comment:
I think we should rename this to `setErrors` since that better describes this method now and update other usages of it to pass an object instead of string. I think the only other usages are in `Form.stories.js` and seems to use [these args](https://github.com/Expensify/App/blob/12314e75dba6488ef7b14e3e51588fa3dc9bc2a8/src/stories/Form.stories.js#L216)